Assets/LeapMotionModules/Interaction/InteractionHand.cs

```diff
@@ -326,7 +326,7 @@ public struct HoverCheckResults {
 
     private const int NUM_FINGERS = 5;
     private const int BONES_PER_FINGER = 3;
-    public const float PER_BONE_MASS_MULTIPLIER = 0.2F;
+    public const float PER_BONE_MASS_MULTIPLIER = 10F;
```

Comment:
Ack this is going to completely change the feel of the interactions in the normal interaction scenes.

It's a shame there's two orders of magnitude difference in how you want to interact with UI vs normal objects...